src/coreComponents/physicsSolvers/NonlinearSolverParameters.hpp

```diff
@@ -246,7 +249,7 @@ class NonlinearSolverParameters : public dataRepository::Group
   /// Flag to apply a line search.
   LineSearchAction m_lineSearchAction;
 
-  /// Flag to pick the type of linesearch
+  /// Flag to pick the type of line search
```

Comment:
are you trying to avoid triggering the line search for the first few iterations?

Reply:
Yes